logger/logger.go

```diff
@@ -616,3 +571,73 @@ func getLabels(data map[string]interface{}) []attribute.KeyValue {
 
 	return attrs
 }
+
+//Log is a basic logging method that fulfills the WagClientLogger interface.
+func (l *Logger) Log(level wcl.LogLevel, title string, m map[string]interface{}) {
+	m["title"] = title
+	l.logWithLevel(LogLevel(level), m)
+}
+
+//NewConcreteLogger creates and returns a concrete logger instead of an interface
+//This was created because New() returned a KayveeLogger instead of a concrete type.
```

Comment:
nit: because these methods are the new standard they shouldn't mention the deprecated ones.
> New creates a *logger.Logger. Default values are Debug LogLevel, kayvee Formatter, and std.err output.

You can just copy the old comments over and mention the concrete vs interface in the deprecation note on the old one.